lib/gitlib.sh

```diff
@@ -21,7 +21,8 @@
 # CONSTANTS
 ###############################################################################
 GITHUB_TOKEN=${FLAGS_github_token:-$GITHUB_TOKEN}
-[[ -n $GITHUB_TOKEN ]] && GITHUB_TOKEN_FLAG=("-u" "$GITHUB_TOKEN:x-oauth-basic")
+GITHUB_TOKEN="$( echo "$GITHUB_TOKEN" | tr -d '[:space:]' )"
```

Comment:
So it was an errant space all along?

Reply:
my guess: trailing newline https://github.com/kubernetes/release/issues/811

Reply:
Really great catch, Hannes!